Gemfile

```diff
@@ -63,6 +63,15 @@ group :development, :test do
   #       thin
   # Hence as dependency with thin is needed we have to install it separately
   gem 'thin', '~> 1.8', '>= 1.8.2'
+
+  # Money-rails: Integrates the Money gem with Rails
+  # Provides a comprehensive solution for handling money and currency conversions
+  gem 'money-rails'
+
+  # OXR: Ruby wrapper for the Open Exchange Rates API
+  # Allows fetching real-time and historical exchange rates
```

Comment:
Generally we don't add comments for commonly used gems but having these extra comments definitely does not harm instead adds code-readability. So we will keep it. Thanks.

Reply:
Sure thanks